### PR TITLE
Correct color naming for accessible combinations

### PR DESCRIPTION
**Pull request by nataliafitzgerald**
## Additions
- Minor content updates to intro text

## Removals
- Removed Black on Gray option for large type (discussed with @caheberer and we agreed that we don't want to encourage the use of this combination as it is difficult to read).

## Changes
- Corrected the color naming for the accessible combinations so that it maps correctly. I was using "green-20" and "green-60" but this didn't map. So, I changed back to "greenmid" and "greentint" (this is in <li class="type-color-combo__"). Since I can use the previous naming convention and still get the right visual result (and correct hex values) I'm moving forward with that approach). 

## Testing

-

## Review

- @caheberer 

[Preview this PR without the whitespace changes](?w=0)

## Screenshots
**Before:** 
screen shot 2017-08-16 at 12 52 16 pm

**After**
screen shot 2017-08-16 at 1 54 28 pm

## Notes
- @caheberer - I moved Gray on White and White on Gray to the accessible combinations since, according to the [color contrast tool](https://accessibility.oit.ncsu.edu/tools/color-contrast/accessible-color-palette.php?&colors=1e9642,20aa3f,66c368,addc91,c7e5b3,e2efd8,005e5d,257675,579695,89b6b5,b4d2d1,d4e7e6,0050b4,0072ce,4497dc,7eb7e8,afd2f2,d6e8fa,002d72,254b87,5674a3,889cc0,b3c0d9,d3daeb,a01b68,b4267a,c55998,d486b2,e3b2cc,f0d8e2,b63014,d14124,dd735d,e79e8e,f0c3b8,f7e0d9,dc731c,ff9e1b,ffb858,ffce8d,ffe1b9,fff0dd,745745,8a6c57,a18573,baa496,d3c5bc,e7ddd7,101820,43484e,5a5d61,75787b,919395,b4b5b6,d2d3d5,e7e8e9,f7f8f9,ffffff&main=ffffff&level=AA) this is now accessible at all sizes. 

## Todos

-

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

**Review by caheberer — APPROVED** [review]

**Review by jenn-franklin — APPROVED** [review]

**Comment by nataliafitzgerald** [comment]
@cfarm - I have approval from @huetingj and @caheberer. Can you merge this when you have a moment?

**Pull request merged**
